internals/overlord/planstate/manager_test.go

```diff
@@ -508,3 +509,82 @@ services:
 
 	c.Assert(calls, Equals, 5)
 }
+
+func (ps *planSuite) TestAppendLayersNoInner(c *C) {
+	plan.RegisterSectionExtension(testField, testExtension{})
+	defer plan.UnregisterSectionExtension(testField)
```

Comment:
The append and combine logic, and the underlying parser logic can be impacted by having an extension registered. Since the content of the file here is none (empty), it may not be strictly necessary, but it does perform the test using the most complex configuration, i.e. with an extension active. I think this is OK, as it mimics the other combine and append unit tests, but I am happy to remove it if you prefer ?

Reply:
No, that's fine -- I just wanted to know the reason.